packages/nova-lib/lib/server/render_context.js

```diff
@@ -31,23 +55,49 @@ const Context = function Context(loginToken) {
   }
 };
 
+export const withRenderContextRaw = (func, options = {}) => {
+  const newFunc = Meteor.bindEnvironment((req, res, next) => {
+    Fiber.current._meteor_dynamics = Fiber.current._meteor_dynamics || [];
+    Fiber.current._meteor_dynamics[DDP._CurrentInvocation.slot] = req.loginContext;
+    Fiber.current._meteor_dynamics[renderContext.slot] = req.renderContext;
+
+    func(req, res, next);
+
+    if (options.autoNext) {
+      next();
```

Comment:
Go to next middleware automatically 🌮, here especially for the `withRenderContext` middleware